pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -1507,7 +1524,11 @@ def add_layer(
         name : str
             Layer name.
         layer_type : str, optional
-            Layer type. Options are ``"S"``, ``"D"``, and ``"G"``. The default is ``"S"``.
+            Layer type. Options are
+            - ``"S"`` for "signal" layer
+            - ``"D"`` for "dielectric" layer
+            - ``"G"`` for "ground" layer
+            The default is ``"S"``.
```

Comment:
```suggestion
            Layer type. The default is ``"S"``. Options are:
            
             - ``"D"`` for "dielectric" layer
             - ``"G"`` for "ground" layer
             - ``"S"`` for "signal" layer

```
Blanks lines needed before and after list items to be formatted correctly in the doc. We try to list the default first and then alphbetize the options.